products/lion/2010_census_tracts/metadata.yml

```diff
@@ -96,6 +97,7 @@ columns:
 - id: the_geom
   name: the_geom
   data_type: geometry
+  description: None
   example: None
   custom:
     readme_data_type: geometry
```

Comment:
If this sub-clause exists for Census Tracts, should it also exist for Census Blocks?

Reply:
and I guess for all other datasets? not that I've looked at them, most others are missing this line